CHANGELOG.md

```diff
@@ -5,8 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.3.0] - 2022-03-01
```

Comment:
Just switch this over to `unreleased` for the date for now. Otherwise git is gonna complain later.